docs/troubleshooting.rst

```diff
@@ -193,3 +193,31 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+Can't override styles using Indigo Theme for MFEs
```

Comment:
I think that this section should be moved to the tutor-indigo README.

Reply:
Shouldn't this also be under Troubleshooting guide? Thinking out loud, this would be the first-place ideally where the people might look to understand what actions would they need to do to override MFEs. I understand Troubleshooting is more related to core-related issues but was wondering that the other items can be useful for the community.